war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -0,0 +1,463 @@
+      dataStorage.updateDocument(LumeerConst.Organization.COLLECTION_NAME, document, organizationIdFilter(organizationId));
+   }
+
+   /**
+    * Reads a list of default roles for newly added users.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @return list of default user roles
+    */
+   public List<String> readDefaultRoles(final String organizationId) {
+      DataDocument defaultRoles = dataStorage.readDocumentIncludeAttrs(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId), Collections.singletonList(LumeerConst.Organization.ATTR_META_DEFAULT_ROLES));
+      return defaultRoles != null ? defaultRoles.getArrayList(LumeerConst.Organization.ATTR_META_DEFAULT_ROLES, String.class) : null;
+   }
+
+   /**
+    * Checks whether the given user has specific role in the organization.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param userName
+    *       name of the user
+    * @param userRole
+    *       name of the specific user role
+    * @return true if the role is assigned to the given user, false otherwise
+    */
+   public boolean hasUserRoleInOrganization(final String organizationId, final String userName, final String userRole) {
+      List<String> userRoles = readUserRoles(organizationId, userName);
+      if (userRoles == null) {
+         return false;
+      }
+      if (userRoles.contains(userRole)) {
+         return true;
+      } else {
+         Map<String, List<String>> roles = userRoleFacade.readRoles(this.getOrganizationId());
+         for (String ur : userRoles) {
+            if (roles.containsKey(ur) && roles.get(ur).contains(userRole)) {
+               return true;
+            }
+         }
+         return false;
+      }
+   }
+
+   private DataFilter organizationNameFilter(final String organizationName) {
+      return dataStorageDialect.fieldValueFilter(LumeerConst.Organization.ATTR_ORG_NAME, organizationName);
+   }
+
+   private DataFilter organizationIdFilter(final String organizationId) {
+      return dataStorageDialect.fieldValueFilter(LumeerConst.Organization.ATTR_ORG_ID, organizationId);
+   }
+
+   private DataFilter userFilter(final String organizationId, final String userName) {
+      Map<String, Object> filter = new HashMap<>();
+      filter.put(LumeerConst.Organization.ATTR_ORG_ID, organizationId);
+      filter.put(dataStorageDialect.concatFields(LumeerConst.Organization.ATTR_USERS, LumeerConst.Organization.ATTR_USERS_USERNAME), userName);
+      return dataStorageDialect.multipleFieldsValueFilter(filter);
+   }
+
+   private DataFilter organizationFilterByUser(final String userName) {
+      Map<String, Object> filter = new HashMap<>();
+      filter.put(dataStorageDialect.concatFields(LumeerConst.Project.ATTR_USERS, LumeerConst.Project.ATTR_USERS_USERNAME), userName);
+      return dataStorageDialect.multipleFieldsValueFilter(filter);
+   }
```

Comment:
@kubedo8 You're right. It does not have to be there. I've already replaced that by `.fieldValueFilter()` method.